jmclient/jmclient/blockchaininterface.py

```diff
@@ -689,7 +692,7 @@ def listunspent(self):
             "amount": u["amount"]
         } for u in self.scan_result["unspents"]]
 
-    def set_wallet_no_history(self, wallet):
+    def set_wallet_no_history(self, wallet) -> None:
```

Comment:
Tried to do that, but ran into problem and don't want to spend too much time just on a single typehint.

It should be `BaseWallet`, which requires moving `INF_HEIGHT` out from blockchaininterface, so that there is no circular imports and we can `from jmclient.wallet import BaseWallet`, ended up with some strange errors in tests, couldn't figure out:
```
self = <test_payjoin.TrialTestPayjoin1 testMethod=test_payment>

    def tearDown(self):
        for dc in reactor.getDelayedCalls():
            dc.cancel()
        res = final_checks(self.wallet_services, self.cj_amount,
>                          self.manager.final_psbt.get_fee(),
                           self.ssb, self.rsb)
E       AttributeError: 'TrialTestPayjoin1' object has no attribute 'manager'

jmclient/test/test_payjoin.py:133: AttributeError
```

And `flake8` doesn't like forward references, which could be an alternative solution.